spdmlib/tests/test_library.rs

```diff
@@ -4,8 +4,7 @@ use codec::{Reader, Writer};
 use spdmlib::common::opaque::*;
 use spdmlib::common::SpdmCodec;
 use spdmlib::config::{
-    MAX_SPDM_CERT_CHAIN_DATA_SIZE, MAX_SPDM_MEASUREMENT_BLOCK_COUNT,
-    MAX_SPDM_MEASUREMENT_VALUE_LEN, MAX_SPDM_OPAQUE_SIZE,
+    MAX_SPDM_CERT_CHAIN_DATA_SIZE, MAX_SPDM_MEASUREMENT_VALUE_LEN, MAX_SPDM_OPAQUE_SIZE,
```

Comment:
We need remove `MAX_SPDM_CERT_CHAIN_DATA_SIZE`.

Reply:
removed!